### PR TITLE
Handle paste event accordingly and strip unwanted characters

### PR DESCRIPTION
**Pull request by schinken**

### CODE REVIEW

**Comment by uhlryk** [comment]
Hi
Thank you for your effort. It looks very useful. I have some small issues.

It also would be great if you write test for this functionality. But if you don't have time I will write tests.

File: src/numberDirective.js [hunk 1]
```diff
@@ -341,6 +345,23 @@
     var append = parameters.append;
 
     var parsedValue = String(value);
+
+    if(wasPasted) {
+      wasPasted = false;
+
+      // Remove all characters which are not number-relevant
+      var regex = new RegExp('[^' + fractionSeparator + thousandSeparator + '0-9]+', 'g');
```
Comment (uhlryk) [inline src/numberDirective.js]: Number can be negative, I think you should add to regex also `-` character.

**Comment by schinken** [comment]
Fixed support for negative numbers. Actually I'm currently not that firm with your test suite and I don't know how to test that paste event.

Would be very nice if you can help me out with the tests, because I also think they're important

**Comment by uhlryk** [comment]
Yes, sure I will add tests. Thanks for your changes.

**Pull request merged**